.azure/pipelines/ci.yml

```diff
@@ -474,17 +474,27 @@ stages:
       jobName: Linux_arm64_build
       jobDisplayName: "Build: Linux ARM64"
       agentOs: Linux
-      buildArgs:
-        --arch arm64
-        --all
-        --pack
-        --no-build-nodejs
-        --no-build-java
-        -p:OnlyPackPlatformSpecificPackages=true
-        -p:AssetManifestFileName=aspnetcore-Linux_arm64.xml
-        $(_BuildArgs)
-        $(_PublishArgs)
-        $(_InternalRuntimeDownloadArgs)
+      steps:
+      - script: ./eng/build.sh
+            --ci
+            --arch arm64
+            --pack
+            --all
+            --no-build-nodejs
+            --no-build-java
+            -p:OnlyPackPlatformSpecificPackages=true
+            -p:AssetManifestFileName=aspnetcore-Linux_arm64.xml
```

Comment:
Official build should break if this setting exists in anything except the last step of the job. You may be :ok: w/o `$(_PublishArgs)` but that just leaves this line as redundant and confusing.

Reply:
Good catch - I copied this from the Linux x64 job, so I guess the duplication exists there too. I'll remove it from both.